release-notes/7.0/7.0.0/api-diff/Microsoft.AspNetCore.App/7.0.0_Microsoft.AspNetCore.Components.Web.md

```diff
@@ -0,0 +1,199 @@
++    [EventHandlerAttribute("onplaying", typeof(EventArgs), true, true)]
++    [EventHandlerAttribute("onpointercancel", typeof(PointerEventArgs), true, true)]
++    [EventHandlerAttribute("onpointerdown", typeof(PointerEventArgs), true, true)]
++    [EventHandlerAttribute("onpointerenter", typeof(PointerEventArgs), true, true)]
++    [EventHandlerAttribute("onpointerleave", typeof(PointerEventArgs), true, true)]
++    [EventHandlerAttribute("onpointerlockchange", typeof(EventArgs), true, true)]
++    [EventHandlerAttribute("onpointerlockerror", typeof(EventArgs), true, true)]
++    [EventHandlerAttribute("onpointermove", typeof(PointerEventArgs), true, true)]
++    [EventHandlerAttribute("onpointerout", typeof(PointerEventArgs), true, true)]
++    [EventHandlerAttribute("onpointerover", typeof(PointerEventArgs), true, true)]
++    [EventHandlerAttribute("onpointerup", typeof(PointerEventArgs), true, true)]
++    [EventHandlerAttribute("onprogress", typeof(ProgressEventArgs), true, true)]
++    [EventHandlerAttribute("onratechange", typeof(EventArgs), true, true)]
++    [EventHandlerAttribute("onreadystatechange", typeof(EventArgs), true, true)]
++    [EventHandlerAttribute("onreset", typeof(EventArgs), true, true)]
++    [EventHandlerAttribute("onscroll", typeof(EventArgs), true, true)]
++    [EventHandlerAttribute("onseeked", typeof(EventArgs), true, true)]
++    [EventHandlerAttribute("onseeking", typeof(EventArgs), true, true)]
++    [EventHandlerAttribute("onselect", typeof(EventArgs), true, true)]
++    [EventHandlerAttribute("onselectionchange", typeof(EventArgs), true, true)]
++    [EventHandlerAttribute("onselectstart", typeof(EventArgs), true, true)]
++    [EventHandlerAttribute("onstalled", typeof(EventArgs), true, true)]
++    [EventHandlerAttribute("onstop", typeof(EventArgs), true, true)]
++    [EventHandlerAttribute("onsubmit", typeof(EventArgs), true, true)]
++    [EventHandlerAttribute("onsuspend", typeof(EventArgs), true, true)]
++    [EventHandlerAttribute("ontimeout", typeof(ProgressEventArgs), true, true)]
++    [EventHandlerAttribute("ontimeupdate", typeof(EventArgs), true, true)]
++    [EventHandlerAttribute("ontoggle", typeof(EventArgs), true, true)]
++    [EventHandlerAttribute("ontouchcancel", typeof(TouchEventArgs), true, true)]
++    [EventHandlerAttribute("ontouchend", typeof(TouchEventArgs), true, true)]
++    [EventHandlerAttribute("ontouchenter", typeof(TouchEventArgs), true, true)]
++    [EventHandlerAttribute("ontouchleave", typeof(TouchEventArgs), true, true)]
++    [EventHandlerAttribute("ontouchmove", typeof(TouchEventArgs), true, true)]
++    [EventHandlerAttribute("ontouchstart", typeof(TouchEventArgs), true, true)]
++    [EventHandlerAttribute("onvolumechange", typeof(EventArgs), true, true)]
++    [EventHandlerAttribute("onwaiting", typeof(EventArgs), true, true)]
++    [EventHandlerAttribute("onwheel", typeof(WheelEventArgs), true, true)]
++    public static class EventHandlers {
```

Comment:
```suggestion
+    public static class EventHandlers {
     }
```

Reply:
Already added it myself. I was faster 😄 .

Reply:
? This is fixing your suggestiong

Reply:
Ah, I see. I missed the closing one.